SPECS/kata-containers-cc/mariner-coco-build-uvm.sh

```diff
@@ -13,7 +13,7 @@ export AGENT_SOURCE_BIN=${SCRIPT_DIR}/kata-agent
 
 # build rootfs
 pushd ${OSBUILDER_DIR}
-sudo make clean
+sudo make DISTRO=cbl-mariner clean
```

Comment:
This should become `DISTRO=azurelinux`

Reply:
On kata-containers, AzureLinux/Mariner is referred to as cbl-mariner.